nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentNotification.java

```diff
@@ -149,9 +164,13 @@ protected void captureAndVerifyCommentEventNotification(Supplier<DocumentModel>
 
             assertEquals(1, handledEvents.size());
 
-            checkDocumentEventContext(handledEvents.get(0), commentDocumentModel, commentedDocumentModel);
-            checkReceivedMail(emailsResult.getMails(), commentDocumentModel, commentedDocumentModel,
+            commentedCommentDocumentModel = commentedCommentDocumentModel == null ? commentedDocumentModel
+                    : commentedCommentDocumentModel;
+
+            checkDocumentEventContext(handledEvents.get(0), commentDocumentModel, commentedCommentDocumentModel);
+            checkReceivedMail(emailsResult.getMails(), commentDocumentModel, commentedCommentDocumentModel,
                     handledEvents.get(0), commentEventType);
+            return commentDocumentModel;
```

Comment:
You don't need to return it.

Reply:
I did this to be able to reply on the comment and prove the document linked in the notification was not the first comment but the top level doc

Reply:
Ok, but it isn't used, no?

Reply:
it was as i was then replying on the returned comment. But I did my own test as you suggested so I changed it